UVtools.Core/Operations/OperationPixelArithmetic.cs

```diff
@@ -646,6 +651,32 @@ protected override bool ExecuteInternally(OperationProgress progress)
                         case PixelArithmeticOperators.DiscardRegion:
                             target.SetTo(EmguExtensions.BlackColor);
                             break;
+                        case PixelArithmeticOperators.Corrode:
+                            {
+                                byte[,] originalData = (byte[,])target.GetData(); // don't need for now, but needed for more subtle noise effects
+                                byte[,] targetData = (byte[,])target.GetData();
+
+                                var zeroMatEntry = new byte[] { 0 }; // add value to ui (or maybe replace by a dimming option)
+                                var noiseThreshold = 280; // add value to ui, values > 255 delete white and increases corrosion intensity
```

Comment:
You can use **_thresholdMaxValue**  here, comes from UI, but 255 limited, to address this use > instead of >=